src/spatial/rect-manager.js

```diff
@@ -27,13 +27,6 @@ Crafty.extend({
            return target;
        },
 
-
```

Comment:
Ah, ok, ignore my previous comment about duplicated functionality.  :)

But I'd actually prefer this code to stay here; I was trying to organize a lot of the methods that were just scattered everywhere in the codebase, and this is moving the opposite direction.  (Probably there are some other functions that should be moved to the rectManager?)

I guess I should also point out that the reason a lot of places use the check explicitly, rather than calling out to a common function, was because theoretically it's a bit faster.  These sort of checks happen deep in the inner loop (they can happen multiple times per entity, per frame) so it's one of the places performance really matters.  That said, I've never actually benchmarked for this explicitly; it's possible the JS engines are smart enough to eliminate that overhead of calling a method.